config_files/metadetect.yaml

```diff
@@ -42,6 +42,23 @@ weight:
 # check for hitting the edge
 bmask_flags: 1073741824  # 2**30
 
+# fraction of slice where STAR or TRAIL was set.  We may cut objects detected
+# there
+star_flags: 96
+
+# we don't interpolate over tapebumps
+tapebump_flags: 16384
+
+# things interpolated using the spline
+spline_interp_flags: 3155
```

Comment:
Are these in the ormask or the bmask? In the bmask this is a fixed value.

Reply:
oh, this is the OR of all flags of things that got interpolated.

Reply:
so I check it in the ormask

Reply:
So would it be better to "or" the bmasks for all bands and then check for the fixed value?

Reply:
Got it. So we are ignoring the coadd bmasks which mark each pixel with the same info. Totally fine by me, just making sure. We will have to make sure to sync up the configs if we change the flags used to build the meds files. 

I do store a copy of the config used to build the meds file in the metadata extension, so this in principle could be automated.

Reply:
I am not sure which is better. Just trying to figure this out. The lines in the APIs are blurring, which always scares me.

Reply:
I would rather use the stuff you already are storing.  Can you help to fix this?

Reply:
I think what you have is best. We cannot have hard coded flags values in metadetect. We write the config once, so that is fine.

Reply:
Yeah I can help - we can have pizza cutter compute the flags on the fly. But it cannot do all of them, only some.

Reply:
all I'm saying is that we could use you values in the bmask rather than re-computing them which I have done, which is error prone and can get out of sync.

I just don't know what those values are

Reply:
OK, maybe we are talking past one another

Reply:
https://github.com/beckermr/pizza-cutter/blob/master/pizza_cutter/des_pizza_cutter/_constants.py#L5